### ISSUE TITLE
Icônes supplémentaires

### ISSUE THREAD

**Issue by razorex** [Kodi-vStream/venom-xbmc-addons#1170]
@LordVenom est-il possible de compléter les icônes avec : anims_views; series_views (copie de views)
films_az

**Comment by TmpName** [comment]
Y a des sites qui les utilisent ?

**Comment by razorex** [comment]
oui certains. series_views de sure. Après à vous de voire si vous faite des icones "homogènes" .
je sais il y a views "tout simplement" mais alors pk films_views? et d'autres..
2 proposition sois vous faites des icones "globaux" views; notes; comments; news etc..... 
ou alors views->films_views. J'espère avoir clairement exposé mon idée.

**Comment by TmpName** [comment]
Yep c'est clair, mais on a le prb de taille de fichier, je pense que a l'inverse @LordVenom va vouloir en supprimer.
Moi perso je suis pr les icones globales, meme si on va se retrouver avec 2 fois les memes icones par exemple si il y a film et serie.

**Comment by razorex** [comment]
ok j'y avait pensé. Après qu'elle taille maxi dois avoir vStream? les deux solutions sont valable sa dépend de l'évolution du skin de vStream.

**Comment by razorex** [comment]
et de toutes façon pour l'instant ce sont les mêmes.

**Comment by LordVenom** [comment]
C'est pas aussi simple, ont n'a des fanart différent pour les series les films ect.. et il faut bien qu'il ce base sur une information pour s'affiche film_news , film_popular le fanart c'est que c'est un films.Aprés ce sont des icones pour les menus afficher en accueil allais mettre un icone dans une source juste pour elle bof

**Comment by LordVenom** [comment]
Je reprend plus précisement pour le moment :

- pour avoir une icone personnel Il faut 4 sites dedans minimum
- il faut que tu ajoute la catégorie a l'accueil ainsi que la fonction qui vas avec.

MOVIE_MOVIE
MOVIE_HD 
MOVIE_VIEWS

C'est ça que j'apelle function si tu organise faut le faire a fond lol

**Comment by LordVenom** [comment]
```
def showAnimes(self):
        oOutputParameterHandler = cOutputParameterHandler()
        oOutputParameterHandler.addParameter('siteUrl', 'ANIM_NEWS')
        oGui.addDir(SITE_IDENTIFIER, 'callpluging', '%s (%s)' % (cConfig().getlanguage(30122), cConfig().getlanguage(30101)), 'animes_news.png', oOutputParameterHandler)

#exempleeeeeeee
        oOutputParameterHandler = cOutputParameterHandler()
        oOutputParameterHandler.addParameter('siteUrl', 'ANIM_VIEWS')
        oGui.addDir(SITE_IDENTIFIER, 'callpluging', '%s (%s)' % (cConfig().getlanguage(30122), cConfig().getlanguage(30101)), 'animes_views.png', oOutputParameterHandler)
```

apelle tous les sites qui ont une Fonction ANIM_VIEWS et affiche l'icone animes_views.png

**Comment by razorex** [comment]
@ LordVenom ok j'ai bien compris pour les icones et les fanart. Mais pour la suite je coince, dans quel fichier faut t'il mettre cette fonction. Dans le fichier "Home"

**Comment by LordVenom** [comment]
Quand tu declarer la variable ANIM_NEWS par exemple et qu'il y a en beaucoup au moin 4 sites different.

Tu peux créer un dossier dans Home oui.

**Comment by razorex** [comment]
j'ai vu des anomalies dans ton fichier "Home" du genre dans showSeries tu appelle l’icône films_hd donc le fanart film
C'est toi qui a défini cette valeur de 4 et est-ce automatique ou je dois vérifier chaque "site"?

**Comment by razorex** [comment]
J'ai testé et donc j'ai ma réponse.
